packages/build-info/src/node/metrics.ts

```diff
@@ -0,0 +1,24 @@
+import { readFile } from 'fs/promises'
+
+import bugsnag from '@bugsnag/js'
+
+import { NoopLogger } from './get-build-info.js'
+
+const { default: Bugsnag } = bugsnag
+const pkgJSON = new URL('../../package.json', import.meta.url)
+
+export async function initializeMetrics(): Promise<void> {
+  try {
+    const { version } = JSON.parse(await readFile(pkgJSON, 'utf-8'))
+    Bugsnag.start({
+      apiKey: process.env.BUGSNAG_KEY_BUILD_INFO || '',
```

Comment:
Should we even try to initialize with a missing key?

Reply:
According to their example, yes: https://github.com/bugsnag/bugsnag-js/blob/next/examples/ts/plain-node/src/app.ts#L4